cdap-cli/src/main/java/co/cask/cdap/cli/commandset/ScheduleCommands.java

```diff
@@ -16,245 +16,41 @@
 
 package co.cask.cdap.cli.commandset;
 
-import co.cask.cdap.api.schedule.Schedule;
-import co.cask.cdap.api.schedule.ScheduleSpecification;
-import co.cask.cdap.cli.CLIConfig;
 import co.cask.cdap.cli.Categorized;
 import co.cask.cdap.cli.CommandCategory;
-import co.cask.cdap.cli.ElementType;
-import co.cask.cdap.cli.english.Article;
-import co.cask.cdap.cli.english.Fragment;
-import co.cask.cdap.cli.exception.CommandInputError;
-import co.cask.cdap.cli.util.AbstractCommand;
-import co.cask.cdap.cli.util.RowMaker;
-import co.cask.cdap.cli.util.table.Table;
-import co.cask.cdap.client.ScheduleClient;
-import co.cask.cdap.internal.schedule.StreamSizeSchedule;
-import co.cask.cdap.internal.schedule.TimeSchedule;
-import co.cask.cdap.proto.id.ScheduleId;
-import co.cask.cdap.proto.id.WorkflowId;
-import co.cask.common.cli.Arguments;
+import co.cask.cdap.cli.command.schedule.AddTimeScheduleCommand;
+import co.cask.cdap.cli.command.schedule.DeleteScheduleCommand;
+import co.cask.cdap.cli.command.schedule.GetScheduleStatusCommand;
+import co.cask.cdap.cli.command.schedule.ListWorkflowSchedulesCommand;
+import co.cask.cdap.cli.command.schedule.ResumeScheduleCommand;
+import co.cask.cdap.cli.command.schedule.SuspendScheduleCommand;
+import co.cask.cdap.cli.command.schedule.UpdateTimeScheduleCommand;
 import co.cask.common.cli.Command;
 import co.cask.common.cli.CommandSet;
 import com.google.common.collect.ImmutableList;
-import com.google.common.collect.Lists;
-import com.google.gson.Gson;
 import com.google.inject.Inject;
 import com.google.inject.Injector;
 
-import java.io.PrintStream;
-import java.util.List;
-
 /**
  * Schedule commands.
  */
 public class ScheduleCommands extends CommandSet<Command> implements Categorized {
-
-  private static final Gson GSON = new Gson();
-
   @Inject
   public ScheduleCommands(Injector injector) {
     super(
       ImmutableList.<Command>builder()
+        .add(injector.getInstance(AddTimeScheduleCommand.class))
+        .add(injector.getInstance(DeleteScheduleCommand.class))
         .add(injector.getInstance(GetScheduleStatusCommand.class))
-        .add(injector.getInstance(SuspendScheduleCommand.class))
-        .add(injector.getInstance(ResumeScheduleCommand.class))
         .add(injector.getInstance(ListWorkflowSchedulesCommand.class))
+        .add(injector.getInstance(ResumeScheduleCommand.class))
+        .add(injector.getInstance(SuspendScheduleCommand.class))
+        .add(injector.getInstance(UpdateTimeScheduleCommand.class))
         .build());
   }
 
   @Override
   public String getCategory() {
     return CommandCategory.APPLICATION_LIFECYCLE.getName();
   }
-
```

Comment:
I assume that all these were simply moved out without modification and did not pay too much attention to detail. If that is not the case, please let me know.